ref/Microsoft.Build.Framework/net/Microsoft.Build.Framework.cs

```diff
@@ -323,6 +323,7 @@ public partial class LazyFormattedBuildEventArgs : Microsoft.Build.Framework.Bui
         protected LazyFormattedBuildEventArgs() { }
         public LazyFormattedBuildEventArgs(string message, string helpKeyword, string senderName) { }
         public LazyFormattedBuildEventArgs(string message, string helpKeyword, string senderName, System.DateTime eventTimestamp, params object[] messageArgs) { }
+        public System.Text.StringBuilder LogOutputProperties { get { throw null; } set { } }
```

Comment:
Exposing a StringBuilder as public API is a bit funky. What's the goal of this new field? Can you get the same output without it (I expected that based on the `(nodeId, contextId)` lookup).

Reply:
I can make it internal and it should still be okay. It's necessary though as it stores the properties that get printed inside the BuildEventargs so it wasn't necessary to change any parameters when printing the properties.